### ISSUE TITLE
論文の Training details にできるだけ寄せたハイパラセットを一度試す

### ISSUE THREAD

**Issue by Uzy03** [Uzy03/TracticAI_SportsData_Competition_2025#11]
lr, dropout, weight decay, batch size, patience など
